docs/advanced.md

```diff
@@ -623,6 +621,21 @@ If you do need to make HTTPS connections to a local server, for example to test
 Response <200 OK>
 ```
 
+### Generate custom SSL Context
+
+Accepts the standard 'verify', 'cert', trust_env' arguments.
+
+```python
+>>> import httpx
+>>> ssl_context = httpx.create_ssl_context()
+```
+
+```python
+ssl_context = httpx.create_ssl_context()
+transport = httpcore.AsyncConnectionPool(ssl_context=ssl_context)
+client = httpx.AsyncClient(transport=transport)
+```
```

Comment:
We haven't yet introduced the "transport" API, so this doesn't feel quite right here.